src/parsec.rs

```diff
@@ -15,25 +15,59 @@ use crate::{
     utils::LogIdent,
 };
 use log::LogLevel;
+use maidsafe_utilities::serialisation;
 #[cfg(not(feature = "mock_parsec"))]
 use parsec as inner;
 use rand::Rng;
-use std::collections::{btree_map::Entry, BTreeMap};
+use std::{
+    collections::{btree_map::Entry, BTreeMap},
+    fmt,
+};
 
 #[cfg(feature = "mock_parsec")]
 pub use crate::mock::parsec::{
-    init_mock, ConsensusMode, NetworkEvent, Observation, Proof, PublicId, SecretId,
+    init_mock, ConsensusMode, Error, NetworkEvent, Observation, Proof, PublicId, SecretId,
 };
 #[cfg(not(feature = "mock_parsec"))]
-pub use parsec::{ConsensusMode, NetworkEvent, Observation, Proof, PublicId, SecretId};
+pub use parsec::{ConsensusMode, Error, NetworkEvent, Observation, Proof, PublicId, SecretId};
 
 pub type Block = inner::Block<chain::NetworkEvent, id::PublicId>;
 pub type Parsec = inner::Parsec<chain::NetworkEvent, FullId>;
 pub type Request = inner::Request<chain::NetworkEvent, id::PublicId>;
 pub type Response = inner::Response<chain::NetworkEvent, id::PublicId>;
 
+// TODO: we'll set PARSEC_SIZE_LIMIT to 1 GB once it's used outside of mock_parsec
+//#[cfg(not(feature = "mock_parsec"))]
+//const PARSEC_SIZE_LIMIT: u64 = 1_000_000_000;
```

Comment:
shall this comment out const be put back?

Reply:
Well it's not used so would have to be under a `allow(unused)` guard?

Reply:
Ah, I think this relates to my comment at https://github.com/maidsafe/routing/pull/1786/files#r326137333
i.e. whether we shall put a triggered empty call